d2l-typography.html

```diff
@@ -8,7 +8,8 @@
 		<style include="d2l-colors d2l-typography-shared-styles">
 
 			.d2l-typography,
-			.vui-typography {
+			.vui-typography,
+			.vui-typography .d2l-body-standard {
```

Comment:
Yeah that's all I meant by not looking right. We no longer write out the `vui-typography` class, so those rules are only there for those who may have been depending on them in their older applications.